docs/en/operations/table_engines/hdfs.md

```diff
@@ -0,0 +1,45 @@
+# HDFS {#table_engines-hdfs}
+
+Manages data on HDFS. This engine is similar
+to the [File](file.md) and [URL](url.md) engine.
+
+## Usage
+
+```
+ENGINE = HDFS(URI, format)
+```
+
+The `format` parameter specifies one of the available file formats. To perform
+`SELECT` queries, the format must be supported for input, and to perform
+`INSERT` queries -- for output. The available formats are listed in the
+[Formats](../../interfaces/formats.md#formats) section.
+
+**Example:**
+
+**1.** Set up the `HDFS_engine_table` table:
+
+``` sql
+CREATE TABLE hdfs_engine_table (name String, value UInt32) ENGINE=HDFS('hdfs://hdfs1:9000/other_storage', 'TSV')
+```
+**2.** Query the data:
+
+``` sql
+SELECT * FROM hdfs_engine_table LIMIT 2
```

Comment:
* Is `INSERT INTO TABLE FUNCTION` supported as well?
* globs example?

Reply:
Globs supported only in table_functions

Reply:
Oops, it works! Amazing